packages/edit-site/src/components/global-styles/style-variations-container.js

```diff
@@ -41,10 +41,30 @@ function Variation( { variation } ) {
 	}, [ variation, base ] );
 
 	const selectVariation = () => {
```

Comment:
this can be tidied up a bit, and needs a test if we decide to go ahead with this.